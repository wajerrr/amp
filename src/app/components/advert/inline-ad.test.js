import React from 'react';
import renderer from 'react-test-renderer';
import InlineAd from './inline-ad';

describe('InlineAd', () => {
  it('should match snapshot', () => {
    const tree = renderer
      .create(
        <InlineAd
          ad={{ siteCode: '1234', zoneCode: 'ABCD', environment: 'dev' }}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
