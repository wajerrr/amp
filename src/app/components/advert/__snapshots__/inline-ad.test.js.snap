// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineAd should match snapshot 1`] = `
<div
  className=""
>
  <amp-ad
    data-slot="/5605/teg.1234/ABCD"
    height="250"
    json="{'targeting': { 'amp': ['y']}}"
    type="doubleclick"
    width="300"
  />
</div>
`;
